ironfish/src/mining/stratum/stratumServer.ts

```diff
@@ -63,8 +63,11 @@ export class StratumServer {
   readonly port: number
   readonly host: string
 
+  readonly maxOpenConnections: number
```

Comment:
You can have more than this number of open connections. This doesn't match the behaviour. In your config it's named, "maxConnectionsPerIp", but here it seems to indicate a global max.